examples/light-client-dapp/src/hooks/useSystemAccount.ts

```diff
@@ -0,0 +1,70 @@
+import { filter, first, map, mergeMap } from "rxjs"
+import { getObservableClient } from "@polkadot-api/client"
+import { ConnectProvider, createClient } from "@polkadot-api/substrate-client"
+import { getDynamicBuilder } from "@polkadot-api/metadata-builders"
+import { useEffect, useState } from "react"
+
+export type SystemAccountStorage = {
+  consumers: number
+  data: {
+    flags: bigint
+    free: bigint
+    frozen: bigint
+    reserved: bigint
+  }
+  nonce: number
+  providers: number
+  sufficients: number
+}
+
+export const useSystemAccount = (
+  provider: ConnectProvider,
+  address: string | null,
+) => {
+  const [balance, setBalance] = useState(0n)
```

Comment:
rename to `systemAccount, setSystemAccount` and return the `storageResult` from L42

Then, the transfer component could do `systemAccount.data.free`